source/MRViewer/MRViewport.cpp

```diff
@@ -65,7 +65,15 @@ void Viewport::shut()
 void Viewport::draw(const VisualObject& obj, const AffineXf3f& xf, bool forceZBuffer, bool alphaSort ) const
 {
     auto modelTemp = Matrix4f( xf );
-    auto normM = ( viewM * modelTemp ).inverse().transposed();
+    auto normTemp = viewM * modelTemp;
+    if ( normTemp.det() == 0 )
+    {
+        auto norm = normTemp.norm();
+        assert( std::isnormal( norm ) );
+        normTemp /= norm;
```

Comment:
We can be here if `norm == 0`, so we should check this before division

Reply:
`std::isnormal` checks for zero either.

Reply:
yes, but `assert` is only in Debug, but we need a check for Release as well

Reply:
Fixed.